src/markdown-pages/add-ons/longhorn.md

```diff
@@ -5,8 +5,17 @@ linktitle: "Longhorn"
 weight: 46
 title: "Longhorn Add-On"
 addOn: "longhorn"
+isDeprecated: true
 ---
 
+## Deprecation Notice
+
+### This add-on is deprecated.
+
+kURL will not be moving forward with support of this add-on in kURL starting March 23, 2023 and will be fully deprecated after September 31st, 2023.  Existing installs that use this add-on will continue to be best-effort supported until the deprecation date.  Please consider using the [OpenEBS](https://kurl.sh/docs/add-ons/openebs) or [Rook](https://kurl.sh/docs/add-ons/rook) add-on for your CSI needs moving forward.  We offer a migration path for existing customer installs as described [here](https://kurl.sh/docs/install-with-kurl/migrating-csi).
```

Comment:
What fully  vs partial deprecated does means? 
It seems that you want say it is deprecated now and at September 31st goes in EOL.

Reply:
Can we use the term remove rather than deprecate?